Remotion/Web/Development.WebTesting.IntegrationTests/TestExecutionScreenshotRecorderTest.cs

```diff
@@ -93,43 +91,6 @@ public void TestExecutionScreenshotRecorderTest_TakeDesktopScreenshot_ReplacesIn
       Assert.That(File.Exists(fullPathWitCharReplaced), Is.True);
     }
 
-    [Test]
-    public void TestExecutionScreenshotRecorderTest_TakeBrowserScreenshot_SavesToCorrectPath ()
```

Comment:
Why is are some of those tests moved to UI-tests. The ScreenshotRecorder basically works, doesn't it? so the moved tests must to something special that's not supported unless there's UI. From the test name it doesn't look like it's UI dependent / different from the other tests. please explain.

Reply:
Note: it's possible that we wanna duplicate the tests (the entire testfixture) because we want to ensure the tests work in both UI and headless mode.

Reply:
I merely moved all tests which didn't work in headless but did work in headful to the new project (hopefully) without changing implementations. I currently can't say what the reason for this is, will look into it tomorrow.

Reply:
Yep, it's stuff noticed with a bit of distance :)